matrix-sdk/src/main/java/org/matrix/androidsdk/MXDataHandler.java

```diff
@@ -2106,4 +2106,8 @@ public RoomIdsListRetroCompat(String aParticipantUserId, String aRoomId) {
 
         return directChatRoomIdsList;
     }
+
+    public RoomsRestClient retrieveRoomRestClient(){
```

Comment:
This is a nice way of reducing this chain. Especially because it is used multiple times in the code.